camayoc/ui/models/components/items_list.py

```diff
@@ -21,15 +21,8 @@ class ItemsList(UIPage):
 
     def _get_item_from_current_list(self, name: str):
         default_timeout = 5000  # 5s
-        name_selectors = [
-            f'div.{class_name}:text-is("{name}")'
-            for class_name in ("list-group-item-heading", "list-item-name")
```

Comment:
Comment: it used to be that depending on a screen, you had to use one of two different class names. I don't know if that's still a case, but I leave the general structure of "check first and return later", just in case it turns out to be still required.